### PR TITLE
screenshots added

### PR DESCRIPTION
**Pull request by vanshg395**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,10 @@
 
 This is an app to send bulk messages to multiple contacts at same time.<br><br>
 This app is built on flutter using <strong>sms</strong> package but it can only be used on an android device as the package used to send message is not supported by iOS.
+
+### Screenshots
+
```
Comment (vinitshahdeo) [inline README.md:7]: You can also use `markdown` syntax.

`sreeenshot`

sreeenshot

**Comment by vanshg395** [comment]
what does this mean?

**Comment by vinitshahdeo** [comment]
@vanshg395 It's markdown syntax for adding images.
**Please refer [this](https://guides.github.com/features/mastering-markdown/)**

**Comment by vanshg395** [comment]
i was unable to resize the image using markdown, thats why i used img tag

**Pull request closed without merging**